SWIPs/swip-honey_to_money.md

```diff
@@ -9,29 +9,29 @@ Created: 31-07-2019
+Swarm needs a mechanism to allow fluctuation of the exchange rate between honey (Swarms internal accounting unit) and the currency used to pay. We propose an on-chain price oracle that returns an `(exchange rate,validUntil)` tuple.  
 
 ## Abstract 
 The ability to update prices is required in order to:
 * Facilitate experimentation with the absolute price of honey (e.g. start at a low price and gradually increase);
-* Off-set fluctuations of the currency used to pay;
-* Lower the price of Swarm in the long-run as the costs of operating a Swarm node decreases.
+* Offset fluctuations of the currency used to pay;
+* Lower the price of Swarm in the long run as the costs of operating a Swarm node decreases.
 
 We propose an on-chain price oracle that returns the current exchange rate when queried. The price-oracle will be initially managed by Swarm developers and stakeholders.
 
 This SWIP is part of a series of SWIPs (but can be implemented on it's own). To see the full picture, please refer to [swip-message_to_honey](./swip-message_to_honey.md), [swip-multiple](./swip-honey_to_money.md), [swip-multiple_payment_processing_support](./swip-multiple_payment_processing_support.md) and the diagram below:
 ![SWIP_Diagrams.svg](./../assets/swip-honey_to_money/SWIP_Diagrams.svg)
 
 ## Motivation
-Nodes keep track of the balances with each other nodes using an imaginary currency: honey. Since honey is not a currency in which nodes can settle their balances with each other - it is merely a unit of account - there is a requirement to facilitate converting honey to a currency. Furthermore, it is desirable that this conversion rate can fluctuate since:
+Nodes keep track of the balances with each other nodes using an internal reference unit: honey. Since honey is not a currency in which nodes can settle their balances with each other - it is merely a unit of account - there is a requirement to facilitate converting honey to a currency. Furthermore, it is desirable that this conversion rate can fluctuate since:
 
 * The Swarm developers plan to initially set the price of honey very low and progressively increase this price to the point where it is sufficiently high to incentivize nodes to offer their service to the network while being low enough not to scare away potential users of Swarm. 
 * We expect the price of the counterpart currency to fluctuate in value against the costs of operating a Swarm node. As the real benefits (i.e. the worth of your compensation) of operating a Swarm node shouldn't fluctuate, the honey price should fluctuate inversely against the fluctuations of currency. If this happens, the real benefit stays equal, regardless of price fluctuations of the currency in which compensation is paid. 
 * The costs of operating a Swarm node is expected to decrease over time, as hardware becomes cheaper, the code more optimized etc. The price of honey is therefore expected to decrease to stay competitive to other storage solutions. 
```

Comment:
"The code more optimized" - I don't think that is necessarily a real trigger to decrease the price. I also wonder how "The price of honey is therefore expected to decrease to stay competitive to other storage solutions." can be justified, it rather sounds like speculation and I would vote to remove this.

Reply:
This is a `long-term` trend, most-likely not noticeable within the next years, but talking about >5 years, it will be definitely noticeable.
Think about it like this: in the end, Swarm will have to compete against other decentralized storage providers, centralized storage providers (Amazon and the like) and even against just storing things on your own hard-drive. Costs of storage has been decreasing over time and for Swarm to stay compatible, honey prices should decrease over time as well.

Reply:
I had similar reaction and we talked about it... so maybe we can consider removing it. The point is, price will change in time, what are the factors that affect it is irrelevant.

Reply:
That would be good for me. Indeed, in the long-term we don't know which factors will be relevant and whether they will have a positive or negative influence. That they will be there, I think, is not a question.